### PR TITLE
docs: Add Icons to Storybook

### PR DESCRIPTION
**Pull request by patrinoua**
The type of this PR is: **docs**

Documentation 

### Add Icons to Storybook 
Screenshot 2021-12-02 at 13 12 30

<details><summary>Changelog updates</summary>

### Changelog updates
`Add Icons to Storybook `

#### Dev changes
Add Icons to Storybook 
-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Add Icons to Storybook)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 01e140d919e58a5d5581da5792a6ba77f0c40c2d
</p>

File: src/palette/icons.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+import * as IconsObject from "./svgs"
+
+storiesOf("Icons", module)
+  .addDecorator(withTheme)
+  .add("icons", () => {
+    const iconKeys = Object.keys(IconsObject)
+    return (
+      <List contentContainerStyle={{ alignItems: "flex-start" }} style={{ paddingLeft: 30 }}>
+        {iconKeys.map((icon) => {
+          // @ts-ignore
```
Comment (pvinis) [inline src/palette/icons.stories.tsx:17]: 👀

File: src/palette/icons.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        })}
+      </List>
+    )
+  })
+
+const styles = StyleSheet.create({
+  contentContainer: { alignItems: "flex-start", paddingLeft: 30 },
+  container: {
+    alignItems: "center",
+    flexDirection: "row",
```
Comment (AlicanAkyuz) [inline src/palette/icons.stories.tsx:34]: **question**: considering that the default direction of a flex container is `row`, do we need to set `flexDirection` explicitly here?

**Review by AlicanAkyuz — APPROVED** [review]
LGTM 🚀  

adding icons to storybook is super useful, I love it!

**Pull request merged**